util/fipstools/acvp/modulewrapper/modulewrapper.cc

```diff
@@ -617,35 +618,41 @@ static bool GetConfig(const Span<const uint8_t> args[],
       R"({
         "algorithm": "RSA",
         "mode": "keyGen",
-        "revision": "FIPS186-4",
+        "revision": "FIPS186-5",
         "infoGeneratedByServer": true,
         "pubExpMode": "fixed",
         "fixedPubExp": "010001",
         "keyFormat": "standard",
         "capabilities": [{
-          "randPQ": "B.3.3",
+          "randPQ": "probable",
```

Comment:
What does this change and that of `primeTest` mean? Did they require any changes to `subprocess/rsa.go`? I may have missed changes to it; all I see is the hex string handling.

Reply:
No changes needed in `rsa.go`, this is a change in how we register our modules capabilities with the ACVP server, and reflects changes to the schema made when register for 186-5 testing.

Thus the values have changed per https://pages.nist.gov/ACVP/draft-celi-acvp-rsa.html#name-keygen-registration-table-2

 In the new schema for FIPS 186-5 `B.3.3` == `probable` and `tblC2` == `2powSecStr`